ext/sysvshm/tests/bug72858.phpt

```diff
@@ -3,7 +3,7 @@ Bug #72858 shm_attach null dereference
 --SKIPIF--
 <?php
 if (!extension_loaded("sysvshm")){ print 'skip'; }
-if (4 < PHP_INT_SIZE) { print "skip 32-bit only"; }
```

Comment:
I belive this condition was simply wrong

Reply:
really? when/where `4 < PHP_INT_SIZE` expr is expected to be true?

Reply:
`4 < 8` is supposed to be true, while `4 < 4` is not. I have to admit that I find Yoda conditions somewhat confusing, but that is no reason to change the test. :)

Reply:
please see https://github.com/php/php-src/runs/6279335088?check_suite_focus=true#step:8:3700

when compiled for x86 (but on Windows x64), the expectation is not present

do you have an access to native Windows 32 bit and can you run the test there?

Reply:
> when compiled for x86 (but on Windows x64), the expectation is not present

The test passes for me in this environment. And no, I haven't got a 32bit Windows machine for more than 10 years.

Reply:
really? for me, when php is built for x86 (and run on x64), the expected warning is not present and thus /wo my change, test is failing

Reply:
If this test no longer works on 32-bit, and there is no way to trigger the same error condition, then it should just be deleted. Your change defeats the purpose of the test.

Reply:
thanks, makes sense!